kratos/solving_strategies/builder_and_solvers/residualbased_elimination_builder_and_solver.h

```diff
@@ -662,9 +662,7 @@ class ResidualBasedEliminationBuilderAndSolver
 
         SizeType nthreads = ParallelUtilities::GetNumThreads();
 
-        typedef std::unordered_set < NodeType::DofType::Pointer, DofPointerHasher>  set_type;
-
-        std::vector<set_type> dofs_aux_list(nthreads);
+        std::vector<DofsArrayType> dofs_aux_list(nthreads);

```

Comment:
you cannot change this ...the insert would be very slow!

i refer to the one on line 690